### DISCUSSION TITLE
404.html does not render results...

### DISCUSSION THREAD

**Discussion by ahfpak** [chipzoller/hugo-clarity#389] [General]
my 404 page is not working properly.

actually, from broken URLs ..it does not redirect to 404.html
Can someone check it for me.. at englishtemplates.com

Thanks in Advance

**Comment by rootwork** [comment]
You're right, it's not.

It looks like you're injecting some kind of ad script in there (and noting that it's being loaded insecurely, as http rather than https) so I can't say from this end what might be going on there, but I don't think it's about Hugo Clarity.
